src/stylesheets/components/_nav.scss

```diff
@@ -2,13 +2,14 @@
 
 .pillbar {
   display: none;
-  
+
   @media ($screen-sm-up) {
     display: flex;
     font-family: $main-font;
     height: 35px;
     margin: 55px auto;
-    width: 40vw;
+    transition: width 0.5s;
+    width: 40%;
```

Comment:
Instead of width 40%, try setting it to full width + `display: flex` + `justify-conent: center`? That should center your pillbar within its container. Then remove the `width` from the `<ul>`.